tools/single_use/fix_double_escape.php

```diff
@@ -152,22 +216,28 @@
 
 
 
-    foreach ($escapedEntries as $instrumentName=>$cids) {
-        printOut("Opening $instrumentName");
+    foreach ($escapedEntries as $tableName=>$cids) {
```

Comment:
@jesscall replacing instrument_name with tablename might be problematic since you need the actual instrument name to instanciate the instrument

Reply:
hmmm, i see... 
the reasoning for using table name in certain cases is very specific to how CCNA has set up Follow-Up instruments. 
I can make adjustments so that `tableName` is used specifically for constructing the queries to `information_schema` and `history`. However, I don't think the logic for doing so is apparent in the Core-LORIS use case, will adding a comment that indicates a project-specific choice be sufficient ?

Reply:
I dont think there are project -specific stuff here. Every LORIS project might be using different instrument and table names. there is no reason here preventing you from associating each instruemnt to a table and use that mapping to instanciate them properly